### PR TITLE
 LPS-205243 Validate friendly url separators configurations

### PR DESCRIPTION
**Pull request by Mikellorza**
subtas: https://liferay.atlassian.net/browse/LPS-205251

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by jkappler** [comment]
@Mikellorza, backend looks good

File: modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx [hunk 1]
```diff
@@ -17,46 +22,65 @@ type Props = {
...
-								defaultValue={field.value}
-								id={field.name}
-								name={field.name}
-							/>
-						</ClayInput.GroupItem>
-					</ClayInput.Group>
-				</ClayForm.Group>
-			))}
+			{fields.map((field) => {
+				const error = errors?.fields?.[field.name];
```
Comment (p2kmgcl) [inline modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx:31]: Are errors or fields optional props? They should be marked as so in the Props type.

File: modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx [hunk 2]
```diff
@@ -17,46 +22,65 @@ type Props = {
...
+									name={field.name}
+								/>
+							</ClayInput.GroupItem>
+						</ClayInput.Group>
+
+						{error ? (
+							<ClayForm.FeedbackGroup>
+								<ClayForm.FeedbackItem>
+									<ClayForm.FeedbackIndicator symbol="exclamation-full" />
+
```
Comment (p2kmgcl) [inline modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx:76]: This extra new line :|

**Review by p2kmgcl — CHANGES_REQUESTED** [review]
@sandrodw3 I just added a couple of tiny comments

**Comment by Mikellorza** [comment]
ci:test:sf

File: modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx:76]: The SF adds it automatically 😔

image

File: modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx:31]: Weeee ✅

File: modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx (hunk 2 above)
Comment (p2kmgcl) [reply modules/apps/friendly-url/friendly-url-web/src/main/resources/META-INF/resources/js/SeparatorFields.tsx:76]: :cry:

**Comment by jkappler** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:forward

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:forward

**Comment by Mikellorza** [comment]
ci:forward:force

**Pull request closed without merging**